cleanlab/internal/multiannotator_utils.py

```diff
@@ -0,0 +1,128 @@
+# Copyright (C) 2017-2022  Cleanlab Inc.
```

Comment:
This file should be added to appear in docs.cleanlab.ai

https://github.com/cleanlab/cleanlab/tree/master/docs/source/cleanlab/internal